src/autocomplete/databases/clickhouse/grammar/ClickHouseParser.g4

```diff
@@ -22,6 +22,7 @@ statements
 statement
     : notInsertStatement (INTO OUTFILE STRING_LITERAL)? (FORMAT identifierOrNull)? (SEMICOLON)?
     | insertStatement
+    | grantStatement
```

Comment:
I need to refactor `noInsertStatement` in the future, current implementation is wrong
https://st.yandex-team.ru/WEBSQL-203